src/test/librbd/journal/test_Entries.cc

```diff
@@ -164,7 +164,6 @@ TEST_F(TestJournalEntries, AioDiscard) {
   REQUIRE_FEATURE(RBD_FEATURE_JOURNALING);
 
   CephContext* cct = reinterpret_cast<CephContext*>(_rados.cct());
-  REQUIRE(!cct->_conf->rbd_skip_partial_discard);
```

Comment:
@dillaman `cct` variable is unused now. But I think we still need to skip the test when partial discard is enabled, due to journal events are optimized out for empty objects. See our discussion here: https://github.com/ceph/ceph/pull/10060#r69556412
```
% CEPH_ARGS='--rbd_skip_partial_discard=true' RBD_FEATURES=109 CEPH_LIB=./lib ./bin/unittest_librbd --gtest_filter=TestJournalEntries.AioDiscard
Note: Google Test filter = TestJournalEntries.AioDiscard
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from TestJournalEntries
[ RUN      ] TestJournalEntries.AioDiscard
/home/mgolub/ceph.ci/src/test/librbd/journal/test_Entries.cc:181: Failure
Value of: wait_for_entries_available(ictx)
  Actual: false
Expected: true
[  FAILED  ] TestJournalEntries.AioDiscard (10018 ms)
[----------] 1 test from TestJournalEntries (10018 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (10047 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] TestJournalEntries.AioDiscard

 1 FAILED TEST
```